Exercises/FizzBuzz.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+
+def isMultipleThree(x):
```

Comment:
The function name isn't that clear, and I think you've forgotten what the functions do.

Each function does the modulus check, x%3 == 0 etc, but you also do that check before you ever call the function. This function could just be `return 'Fizz'`, at which point it doesn't need to be a function.

But pulling the logic out into functions is the correct approach. At the moment your code for rule 13 is pretty complicated and will get more complicated whenever new rules are added, or if it's possible for some rules to be skipped.
Your rule 17 logic also has to be changed if new rules get added that might make the result longer, and would entirely break if there were any 3 letter words.

Can you see a simpler approach? Define a functions, where you could just apply one function after another? i.e.
```
output = multiple_3_rule(current_number, output)
output = multiple_7_rule(current_number, output)
etc
print(output)
```

Or more succinctly:
```
output = current_number
for rule in rules:
    output = rule(current_number, output)
print(output)
```